maven-embedder/src/main/java/org/apache/maven/cli/transfer/AbstractMavenTransferListener.java

```diff
@@ -184,14 +189,19 @@ protected AbstractMavenTransferListener(PrintStream out) {
 
     @Override
     public void transferInitiated(TransferEvent event) {
+        String darkOn = MessageUtils.isColorEnabled() ? ANSI_DARK_SET : "";
```

Comment:
If we want to cut logging (and I do agree with @elharo we log way too much), I'd just remove this method (make it no op). 

Or just leave it for PUT (leave it as before), and do not log nothing for the rest.

Reason is simple: if you have 5 reposes, you will have 5 "Downloading...." but only one "Downloaded", and the latter matter.

Reply:
This should be a separate discussion. Foe this we have other listeners.

Reply:
this line is not about cutting logging: it's about adding color when color enabled

Reply:
Well JIRA currently states following:

>transfer message are currently hard to read for many users
> (logging example cut for brevity)
> * it's interweaved into normal build messages
> * users don't really see the difference between "Downloading" (transfert started, may eventually fail with 404) and "Downloaded" (done successfully)
> * repository id is not so visible in the middle of the message
> * the download url has much info in it to see: base url, groupId as directory, artifactId, version, and filename

So, based on this above, I assumed, that: is interweave and too chatty, so cut it down, and if "users don't see the difference", they are right: As I said, this logger when 5 repositories are defined will emit 5 "Downloading" and 1 "Downloaded", and the one "Downloaded" matters, the 5 "Downloading" does not (better check with effective-pom what reposes you have).

Unsure how JIRA comes to conclusion based on these 4 bullets above "ok, let's color it!", when it is ONLY solving the last bullet.

Reply:
> This should be a separate discussion. Foe this we have other listeners.

What other listeners we have for this? WDYM?

Reply:
As we know, by default Maven "leaks" your artifact paths, and **only reason** "Downloading" log line is useful is only this one: it makes user **aware** Maven is making HTTP request against the logged baseUrl. OTOH, due overwhelming amount of Maven logging, this information is usually lost, due log-noise.

To properly fix "leaking", use [RRF](https://maven.apache.org/resolver/remote-repository-filtering.html) but that is another thing.